app/views/documents/edit.html.erb

```diff
@@ -33,14 +32,32 @@
 
 <script src="https://js.shubox.io/v1/<%= Rails.application.secrets.shubox_js_id %>.js"></script>
 
+<script id="template" type="text/x-handlebars-template">
+<%= render "document", url: nil %>
```

Comment:
This is in order to trigger the usage of the handlebars `"{{variable}}"` inside the partial. Like:

> "Url is nil so expect that this will be used as a handlebars template - spit this out with the `'{{...}}'` placeholders."